fiftyone/utils/sharp.py

```
@@ -0,0 +1,213 @@
+    Args:
+        config: a :class:`AppleSharpModelConfig`
+    """
+
+    def __init__(self, config):
+        super().__init__(config)
+        self.config.raw_inputs = True
+        self._output_dir = config.output_dir
+        self._focal_length_mm = config.focal_length_mm
+        self._output_dir_initialized = False
+
+    def _load_model(self, config):
```

Ruff (0.14.14)

Missing return type annotation for private function `_load_model`

(ANN202)